README.rst

```diff
@@ -60,11 +60,13 @@ Development
 
     pip install -r requirements/default.txt
 
-**Step 3.** Install fury via::
+**Step 3.** Install fury 
+
+As `local project install <https://pip.pypa.io/en/stable/reference/pip_install/#id44>`__ using::
 
     pip install .
 
-or::
+Or as `"editable" installs <https://pip.pypa.io/en/stable/reference/pip_install/#id44>`__ using::
```

Comment:
Or as an "editable" installation

Also say something like "If you are developing FURY, you most likely want this option."

Reply:
Did that, please take a look !